### ISSUE TITLE
Is DllExport not supported outside of Windows OS and/or Visual Studio?

### ISSUE THREAD

**Issue by wixyer** [3F/DllExport#63]
With v1.6 release, now I have a tricky question.
You see, "Batch" (`*.bat`) file is Windows-only version of command-line scripting.
[Visual Studio starts to support macOS](https://blogs.msdn.microsoft.com/visualstudio/2016/11/16/visual-studio-for-mac/).
(Windows and macOS are very different in their nature.) If we also add Linux to that (since it is also capable of compiling and running C# code, because thanks to Mono & .NET Core), we have all 3 major operating systems of today, and all of them can be used to develop C# (they all have IDE as well).

Since I highly doubt that this is actually working on all 3 operating systems, my question is the following:
**Is DllExport (latest version) cross-platform?**
In other words, **does DllExport support all 3 major operating systems?**

(If it is supported outside of Windows, please answer "how so?")

p.s. Happy New Year!

**Comment by wixyer** [comment]
Possible duplicate of #45.
Let's focus on cross-platform support on this one?

**Comment by 3F** [comment]
> **With v1.6 release**, now I have a tricky question.

1.6 provides same features as for 1.5.x versions. We did not break any compatibility from this.

* https://github.com/3F/DllExport/blob/master/changelog.txt

However, now we finally changed distribution of packages with adding of embeddable manager.

DllExport.**bat** mainly is a wrapper that's based on [GetNuTool core](https://github.com/3F/GetNuTool). Finally it just aggregates calling to Wizard.


```bash
msbuild gnt.core /p:ngpackages="DllExport"
```
 v
```bash
msbuild net.r_eg.DllExport.Wizard.targets ...
```

All around msbuild and we just need to add same an wrapper for bash (it's much easier [than it was for batch](https://3f.github.io/DllExport/releases/latest/manager/)), or simply call it manually via GetNuTool & msbuild etc.

**But, what about cross-platforms today**:  -> https://github.com/3F/DllExport/issues/17#issuecomment-335001440

> p.s. Happy New Year!

:) Happy New Year ! and thanks for using !
